pkg/cedar/core/system_summary.go

```diff
@@ -26,9 +36,15 @@ func (c *Client) GetSystemSummary(ctx context.Context, tryCache bool) ([]*models
 
 	// Check and use cache before making API call
 	if tryCache {
-		cachedSystems, found := c.cache.Get(systemSummaryCacheKey)
-		if found {
-			return cachedSystems.([]*models.CedarSystem), nil
+		cachedSystemMap := c.getCachedSystemMap(ctx)
+		if cachedSystemMap != nil {
+			cachedSystems := make([]*models.CedarSystem, 0, len(cachedSystemMap))
+
+			for _, sys := range cachedSystemMap {
+				cachedSystems = append(cachedSystems, sys)
```

Comment:
Since you're using `make` with `len(cachedSystemMap)`, I think you can actually just directly set the `cachedSystems` at an index, rather than using `append`

(To be fair, this is a nit - just a possible minor performance bump)

Reply:
To do that, you'd need to change the `0` to `len(cachedSystemMap)` as well, or just change it to `make([]*models.CedarSystem, len(cachedSystemMap))`. In the three-argument form of `make` with slices, the third argument is the capacity, not the length.

Reply:
@ClayBenson94 @DylanSpOddball I made this change, but on further thought, I believe the way I had it originally is essentially equivalent to setting a specific index. With the third argument of make, I'm giving it a capacity, so it calls malloc once, allocates space for the number of items given, and append is just tracking the memory offset of the next element, no?

Reply:
@jstrang-oddball That's _probably_ the case, depending on how Go implements slices under the hood. Ultimately, I think the two-argument form that you're using now is clearer, though.